Website/src/Foundation/BranchPresets/AddFromBranchPreset.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Linq;
+using Sitecore.Data.Items;
+using Sitecore.Diagnostics;
+using Sitecore.Pipelines.ItemProvider.AddFromTemplate;
+using Sitecore.StringExtensions;
+
+namespace Sitecore.Foundation.BranchPresets
```

Comment:
I know this code is coming from OSS.

Is there any part you customized and is not a direct copy? If yes, which parts so I can review those?

I have also 2 asks:
- Can we update the namespace according to the chosen assembly name?
- Can we add a comment to tell where we copied that code/config from? That would be very kind to the author of that solution.

Thanks

Reply:
Added info to AssemblyInfo